packages/docs/docker.package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "docs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "npm run generate:components:docs && next dev",
+    "build": "npm run generate:components:docs && next build",
+    "start": "next start -H 0.0.0.0 -p ${PORT:-3000}",
+    "generate:components": "node docgen.js",
+    "generate:components:docs": "npm run generate:components && node docs-generator.js",
+    "generate:watch": "concurrently \"npm run generate:components -- --watch\" \"npm run generate:components:docs -- --watch\"",
+    "serve": "node server.js"
+  },
+  "dependencies": {
+    "@patternfly/react-core": "^4.79.2",
+    "@patternfly/react-icons": "^4.7.18",
+    "@patternfly/react-table": "^4.19.24",
+    "@redhat-cloud-services/frontend-components": "^3.0.10",
+    "classnames": "^2.2.6",
+    "next": "^10.0.7",
+    "prism-react-renderer": "^1.1.1",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-json-view": "^1.19.1",
+    "react-redux": "^7.2.2",
+    "react-router-dom": "^5.2.0",
+    "redux": "^4.0.5"
+  },
+  "devDependencies": {
```

Comment:
I don't think there is a difference between devDeps and deps. 🤔  Maybe we could merge them together.

Reply:
Nah keeping it separate is good for our sanity.